pages/api/profiles/[username]/embed.js

```diff
@@ -0,0 +1,46 @@
+import satori from "satori";
+import path from "node:path";
+import fs from "node:fs/promises";
+import { fileURLToPath } from "node:url";
+import { getUserApi } from "./index";
+import Profile from "@components/embeds/external/Profile";
+import { loadEmoji, getIconCode } from "@services/utils/twemoji";
+
+const inter = await fs.readFile(path.resolve(path.dirname(fileURLToPath(import.meta.url)), "../../../../public", "Inter-Regular.ttf"));
+const interBold = await fs.readFile(path.resolve(path.dirname(fileURLToPath(import.meta.url)), "../../../../public", "Inter-Bold.ttf"));
+
+export default async function handler(req, res) {
+  const { username } = req.query;
+  const { profile } = await getUserApi(req, res, username);
+
+  const response = await satori(
+    <Profile data={profile} />,
+    {
+      width: 350,
+      height: 550,
+      fonts: [
+        {
+          name: "Inter",
+          data: inter,
+          weight: 400,
+          style: "normal",
+        },
+        {
+          name: "Inter",
+          data: interBold,
+          weight: 700,
+          style: "normal",
+        },
+      ],
+      loadAdditionalAsset: async (code, segment) => {
+        if (code === 'emoji') {
+          return (`data:image/svg+xml;base64,${btoa(await loadEmoji("twemoji", getIconCode(segment)))}`)
+        }
+        return;
```

Comment:
Do you think that this return could be removed ( I am not sure about that though)

Reply:
Haha 😀

I didn't try that yet, but I guess it can be removed though since the logic makes it to the last line there it will leave the block anyway.

I'll check it out though.

Reply:
Awesome, thank you for that! let me know about the result of your investigation, I might need to update the open sauced code then lol.

Reply:
So, I removed the `return` and it turns out it had no effect on the logic atleast I can say that for my local development environment.

You might wanna give it a try too! 😉